integration/kubernetes/runtimeclass_workloads/job.yaml

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: job-pi-test
+spec:
+  template:
+    spec:
+      runtimeClassName: kata
+      containers:
+      - name: pi
+        image: perl
+        command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(10)"]
```

Comment:
Wow. I'd forgotten `perl` was still a "thing" ;) Not a blocker but this might look a little less crazy using `python`:

```
python -c "import math; print math.pi"
```